src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -47,10 +50,35 @@
 #include "ui_automatedrssdownloader.h"
 #include "automatedrssdownloader.h"
 
+class AutomatedRssDownloader::DownloadRuleListMatchState
+{
+public:
+    DownloadRuleListMatchState();
+    ~DownloadRuleListMatchState();
+    void clear();
+
+public:
+    QSet<QPair<QString, QString >> m_treeListEntries;
+};
+
+AutomatedRssDownloader::DownloadRuleListMatchState::DownloadRuleListMatchState()
+{
+}
+
+AutomatedRssDownloader::DownloadRuleListMatchState::~DownloadRuleListMatchState()
+{
+}
+
+void AutomatedRssDownloader::DownloadRuleListMatchState::clear()
+{
+    m_treeListEntries.clear();
+}
+
 AutomatedRssDownloader::AutomatedRssDownloader(const QWeakPointer<Rss::Manager> &manager, QWidget *parent)
     : QDialog(parent),
     ui(new Ui::AutomatedRssDownloader),
-    m_manager(manager), m_editedRule(0)
+    m_manager(manager), m_editedRule(0),
+    m_ruleMatcher(new AutomatedRssDownloader::DownloadRuleListMatchState())
```

Comment:
Something went wrong with uncrustify.
The initialization list should look like this:
```c++
AutomatedRssDownloader::AutomatedRssDownloader(const QWeakPointer<Rss::Manager> &manager, QWidget *parent)
     : QDialog(parent)
     , ui(new Ui::AutomatedRssDownloader)
     , m_manager(manager)
     , m_editedRule(0)
     , m_ruleMatcher(new AutomatedRssDownloader::DownloadRuleListMatchState())
{
}
```